src/types/WidthPercent.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable @typescript-eslint/no-magic-numbers */
+
+/**
+ * Values pulled from cfpb-core width utilities
+ * https://github.com/cfpb/design-system/blob/main/packages/cfpb-core/src/utilities.less#L328-L386
+ */
+export type WidthPercent =
```

Comment:
It's kind of a bummer we can't import these out of the design system. 🤔 I think we could refactor some DS stuff to make that work... but probably for another day. 👍